replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorImpl.java

```diff
@@ -311,11 +288,11 @@ public void cancelSyncRequest(String configId) {
     return Collections.unmodifiableSet(new HashSet<>(activeSyncRequests));
   }
 
-  private ReplicationStore getStoreForId(String siteId) {
-    ReplicationStore store;
+  private NodeAdapter getStoreForId(String siteId) {
+    NodeAdapter store;
     ReplicationSite site = siteManager.get(siteId);
     try {
-      store = replicatorStoreFactory.createReplicatorStore(new URL(site.getUrl()));
+      store = nodeAdapters.factoryFor(NodeAdapterType.DDF).create(new URL(site.getUrl()));
```

Comment:
I think we want the factory to figure out which kind of adapter we need. Soon when we have another type of adapter we will not be able to do this. I think we want the factory to figure out from the url (maybe by even creating a client and attempting a ping) which adapter type to use.

Reply:
Or we could update the sites object to have some sort of type that the user selects during creation.